### DISCUSSION TITLE
BD Rhapsody + Nanopore Sequencing

### DISCUSSION THREAD

**Discussion by skyng22003** [milaboratory/mixcr#1709] [Q&A]
Would it be possible to modify the "generic-ont-with-umi" protocol to include the BD Rhapsody Cell barcodes for TCR / mRNA analysis?

Thanks,
Sky

**Comment by mizraelson** [comment]
Hi, which BD protocol did you use to generate the data? Also do you have ONT adapters trimmed from the FASTQ reads? I can create a custom preset for your dataset but it would help if you can share a subsample of reads from the original fastq file for me to test. You can write us at support@milaboratories.com.

**Reply by skyng22003** [reply]
Thank you for replying, I have written to support@milaboratories.com with the details.
